src/iter/mod.rs

```diff
@@ -173,6 +173,17 @@ pub trait ParallelIterator: Sized + Send {
     type Item: Send;
 
     /// Executes `OP` on each item produced by the iterator, in parallel.
+    /// 
+    /// ```
+    /// use std::sync::mpsc::channel;
+    /// use rayon::prelude::*;
+    /// let (tx, rx) = channel();
+    /// (0..5).map(|x| x * 2 + 1).par_iter()
```

Comment:
`(0..5).into_par_iter().map(...)`. Right now there's no way to obtain a `ParallelIterator` from a general `Iterator` like `map` returns.